pkg/ui/react-app/package.json

```diff
@@ -45,7 +45,8 @@
     "tempusdominus-bootstrap-4": "^5.1.2",
     "tempusdominus-core": "^5.0.3",
     "typescript": "^3.3.3",
-    "use-query-params": "^1.1.6"
+    "use-query-params": "^1.1.6",
+    "y18n": "4.0.1"
```

Comment:
`y18n` is not a direct dependency so adding it explicitly will create another copy of it with different version. You can see that now we have two entries for `y18n` in `yarn.lock`.

You can see which exact module to update if you run `yarn why y18n`.

```
yarn why y18n
yarn why v1.22.10
[1/4] Why do we have the module "y18n"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "y18n@4.0.0"
info Reasons this module exists
   - "yargs" depends on it
   - Hoisted from "yargs#y18n"
   - Hoisted from "react-scripts#webpack-dev-server#yargs#y18n"
   - Hoisted from "react-scripts#webpack#terser-webpack-plugin#cacache#y18n"
info Disk size without dependencies: "24KB"
info Disk size with unique dependencies: "24KB"
info Disk size with transitive dependencies: "24KB"
info Number of shared dependencies: 0
Done in 0.56s.
```
I ran it locally and it looks like we need to upgrade `react-scripts` to the latest minor version.

Also, I think it'd be better to upgrade all dependencies to the latest patch version as well by running `yarn upgrade` or `yarn upgrade-interactive`.